app/models/subscriber.rb

```diff
@@ -0,0 +1,3 @@
+class Subscriber < ActiveRecord::Base
+
+end
```

Comment:
Would it be worth adding validations or shall we do that later?

e.g. unique address (case insensitive), matches an email regex(?)

Reply:
I intentionally left that off as it wasn't required to complete the ticket and we're not accepting user input yet. Seemed like that would be something to think about then or when we're looking at the data we need to migrate.

Reply:
[Trello](https://trello.com/c/BhKdFEXN/170-subscriber-validation)